doc/Edit contributing.rst (#29)
pyansys/dev-guide: doc/source/overview/contributing.rst

Starting Discussions
--------------------
For general questions about development practices, you should create discussions
rather than issues. The repository for each PyAnsys library has its own

> ``Discussions`` page. For example, to ask a question about an PyMAPDL development

Author: ```suggestion
``Discussions`` page. For example, to ask a question about a PyMAPDL development
```

    cd pymapdl
    pip install -e .

If you want to eventually push a contribution to the official
PyAnsys repository, consider creating a `fork <https://docs.github.com/en/get-started/quickstart/fork-a-repo>`_
of the repository. For additional information, see :ref: dev_practices.

Licensing
---------
All contributed code will be licensed under the MIT License. For more information, see

> for more information. The ``LICENSE`` file with the MIT License must be included in

Author: ```suggestion
:ref:`license_file`. The ``LICENSE`` file with the MIT License must be included in
```
